lib/src/conversion.dart

```diff
@@ -57,21 +59,19 @@ Color hslToRgb(HslColor hsl) {
     b = _hue2rgb(p, q, h - 1 / 3);
   }
   return Color.fromARGB(
-      hsl.a.round(), (r * 255).round(), (g * 255).round(), (b * 255).round());
-}
-
-HSVColor colorToHsv(Color color) {
-  return HSVColor.fromColor(color);
+    hsl.a.round(),
+    (r * 255).round(),
+    (g * 255).round(),
+    (b * 255).round(),
+  );
 }
 
-HSVColor rgbToHsv(
-    {required int r, required int g, required int b, required int a}) {
-  return colorToHsv(Color.fromARGB(a, r, g, b));
-}
-
-Color hsvToColor(HSVColor hsv) {
-  return hsv.toColor();
-}
```

Comment:
Has this been removed on purpose?

Reply:
Yes because it is being used at one place only.